dom.bs

```diff
@@ -5154,6 +5170,11 @@ invoked, must run these steps:
 
  <li><p>Initialize <var>event</var>'s {{Event/type}} attribute to the empty string.
 
+ <li><p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to a {{DOMHighResTimeStamp}}
+ which is the high resolution time from the <a>time origin</a> to the occurrence of the call to
+ {{Document/createEvent()}}.
+
+
```

Comment:
Done. 

As for duplicate initialization, I am not sure if I understand. The way I was reading the spec was that for non-synthetic events we do the `2.4 - concept-event-create ` that does not require invoking `2.4. - concept-event-constructor` which is only needed for synthetic events.